src/Components/QuestionListReversedStrings/utils.ts

```diff
@@ -0,0 +1,33 @@
+import { Question } from "../../Apis/types";
+
+interface Props {
+    questions: Question[];
+}
+
+const reverseOddWordsInSentence = (string: string) => {
+    let newString = string.split(" ");
+    return newString
+        .map((word) => {
+            if (word.length % 2 !== 0) {
+                word = word.split("").reverse().join("");
```

Comment:
no reverse function of a string only for arrays, but I could do a swap function with two pointers

Reply:
What I'm saying is that for practice it would be good to write a reverse function instead of using one that exists. Could be good to try to write it as many different ways as you can think of. Get some practice in on trying to think past your first solution.